components/Containers/MetaBar/index.module.css

```
@@ -18,6 +18,8 @@
     xs:border-t-neutral-200
     xs:dark:border-t-neutral-900;
 
+  overflow-wrap: anywhere;
+
```

You can use the following example for classes that do not have an equivalent in Tailwind;

```suggestion
    xs:dark:border-t-neutral-900
    [overflow-wrap:anywhere];

```